NOVAthesisFiles/Schools/uminho/univ-defaults.ldf

```diff
@@ -93,10 +93,21 @@
 %------------------------------------------------------------------
 
 %%%%%%%%% Important package for the licensing. 
+
+\assocarray{doclicenseLanguage}
+\doclicenseLanguage(pt):={portuguese}
+\doclicenseLanguage(en):={english}
+\doclicenseLanguage(fr):={french}
+\doclicenseLanguage(it):={italian}
+\doclicenseLanguage(de):={german}
+\doclicenseLanguage(es):={spanish}
```

Comment:
This is definitely the wrong approach.  Just replace it with:
\options{/@nt/langshorttolong=\option{/novathesis/copyrightlang}}